src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LogoWrapper } from './Logo.styles';
+
+const Logo = ({ color }) => {
```

Comment:
Add a default value to color:
```
const Logo = ({ color='red' }) => {
...
}
```

Reply:
Thank you for review. I have done changes.